dashboard/app/views/pd/workshop_enrollment/cancel.html.haml

```diff
@@ -0,0 +1,8 @@
+%h1
+  Your registration has been canceled.
```

Comment:
fwiw our codebase seems to use cancelled more than canceled, though i think both are correct :)